README.md

```diff
@@ -130,7 +130,8 @@ Either way it provides access to the following fields:
 * **annotations** Annotations to apply to the pod.
 * **inheritFrom** List of one or more pod templates to inherit from *(more details below)*.
 * **slaveConnectTimeout** Timeout in seconds for an agent to be online.
-* **activeDeadlineSeconds** Pod is deleted after this deadline is passed.
+* **podRetention** Controls the behavior of keeping slave pods. Can be 'NEVER', 'ON_FAILURE', or 'ALWAYS' - if empty will default to the plugin's "Pod Retention" setting.
```

Comment:
might be clearer to say `... if empty will default to deleting the pod after `activeDeadlineSeconds` has passed.`

Reply:
that's true if the cloud's setting isn't changed (which, granted, will cover most cases)

Reply:
yep ... and mentioning the "inherit from the cloud setting" should be mentioned somehow in this doc as well I would think ... so combine the two pieces here